sdk/schemaregistry/Azure.Data.SchemaRegistry/src/autorest.md

```diff
@@ -6,3 +6,37 @@ Run `dotnet build /t:GenerateCode` to generate code.
 title: SchemaRegistryClient
 require: https://github.com/Azure/azure-rest-api-specs/blob/7aefe11d56a0b48615b0ab01cad566d6c39a5b08/specification/schemaregistry/data-plane/readme.md
 ```
+
+## Swagger workarounds
```

Comment:
Is this specific to .NET?  Or a point in time thing?  Can we fix this in the actual swagger?

Reply:
it is not specific for .NET. The swagger guidelines prohibits content-type to be defined explicitly. The swagger has `produces` and `consumes` properties instead for that purpose. However, in this specific case, we need to allow the customer to influence the content-type header values so we wanted it to be an explicit parameter in the generated method.